components/tooltip/tooltip.js

```diff
@@ -650,6 +657,27 @@ class Tooltip extends RtlMixin(LitElement) {
 		return spaceAround;
 	}
 
+	async _determineIfTruncating() {
+		// if no resize has happened since truncation was previously calculated the result will not have changed
+		if (!this._resizeRunSinceTruncationCheck || !this.onlyShowIfTruncating) return;
+
+		const target = this._target;
+		const clone = target.cloneNode(true);
+		clone.removeAttribute('id');
```

Comment:
Curious if you ran into a problem with the id being present on the clone, or is this just a careful measure?

Reply:
@svanherk not sure if you had thoughts on this but I believe it's just a careful measure - in practice the element wouldn't be around long enough to cause problems